kansha/app/comp.py

```diff
@@ -46,11 +46,11 @@ class Kansha(object):
     """The Kansha root component"""
 
     def __init__(self, app_title, app_banner, theme,
-                 search, services_service):
+                 card_extensions, search, services_service):
         """Initialization
         """
         self._services = services_service
-
+        self.card_extensions = card_extensions
         self.app_title = app_title
         self.app_banner = app_banner
         self.theme = theme
```

Comment:
Set the instance vars in the same order then the parameters (here `self.app_title`, `self.app_banner`, `self.theme`, `self.card_extensions`, `self.search`, `self.services`)